src/adapter/src/catalog/storage.rs

```diff
@@ -1323,6 +1323,10 @@ impl<'a, S: Append> Transaction<'a, S> {
         }
     }
 
+    /// Removes item `id` from the transaction.
```

Comment:
Since you're updating the docs, can you document here and below that an error is returned if any `id` is not found?

Reply:
Updated.